Forge.Forms/src/Forge.Forms/DynamicExpressions/MultiValueConverters/DivideMultiConverter.cs

```diff
@@ -9,80 +9,86 @@ public class DivideMultiConverter : IMultiValueConverter
-                            case long l2: return dc1 / l2;
-                            case int i2: return dc1 / i2;
-                            case short s2: return dc1 / s2;
-                        }
-                        break;
-                    case double d1:
-                        switch (values[1])
-                        {
-                            case decimal dc2: return (decimal)d1 / dc2;
-                            case double d2: return d1 / d2;
-                            case float f2: return d1 / f2;
-                            case long l2: return d1 / l2;
-                            case int i2: return d1 / i2;
-                            case short s2: return d1 / s2;
-                        }
-                        break;
-                    case float f1:
-                        switch (values[1])
-                        {
-                            case decimal dc2: return (decimal)f1 / dc2;
-                            case double d2: return f1 / d2;
-                            case float f2: return f1 / f2;
-                            case long l2: return f1 / l2;
-                            case int i2: return f1 / i2;
-                            case short s2: return f1 / s2;
-                        }
-                        break;
-                    case long l1:
-                        switch (values[1])
-                        {
-                            case decimal dc2: return l1 / dc2;
-                            case double d2: return l1 / d2;
-                            case float f2: return l1 / f2;
-                            case long l2: return l1 / l2;
-                            case int i2: return l1 / i2;
-                            case short s2: return l1 / s2;
-                        }
-                        break;
-                    case int i1:
-                        switch (values[1])
-                        {
-                            case decimal dc2: return i1 / dc2;
-                            case double d2: return i1 / d2;
-                            case float f2: return i1 / f2;
-                            case long l2: return i1 / l2;
-                            case int i2: return i1 / i2;
-                            case short s2: return i1 / s2;
-                        }
-                        break;
-                    case short s1:
-                        switch (values[1])
-                        {
-                            case decimal dc2: return s1 / dc2;
-                            case double d2: return s1 / d2;
-                            case float f2: return s1 / f2;
-                            case long l2: return s1 / l2;
-                            case int i2: return s1 / i2;
-                            case short s2: return s1 / s2;
-                        }
-                        break;
+                    switch (values[0])
```

Comment:
Could we move these converters to demo and inject them from outside?
I'd prefer to ship a more flexible converter for math in the future. Ideally we'd support any kind of mathematical expressions to make it easy doing this from xml.